realm/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -173,10 +173,10 @@ public Boolean call() throws Exception {
         while (!isReady.get()) {
             Thread.sleep(5);
         }
-        Thread.sleep(100); 
+        Thread.sleep(100);
```

Comment:
In this case we do care though as that would be a test failure. Also as we just add ```throws InterruptedException``` to the method name it doesn't really effect readability of the test.